dist/formats/gone/publisher/schema.json

```diff
@@ -43,12 +43,33 @@
           }
         }
       }
+    },
+    "links": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "can_be_replaced_by": {
+          "description": "The optional content_item that this gone item has replaced.",
```

Comment:
Would this read better as: `Optional. The content_item that this gone item has replaced.`?